tests/EventStream.Tests/EventDispatcherTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using EventStream.Tests.Fakes;
+using FluentAssertions;
+using Microsoft.Framework.DependencyInjection;
+using System;
+using System.Linq;
+using Xunit;
+
+namespace EventStream.Tests
+{
+    public class EventDispatcherTests
+    {
+        static IServiceProvider CreateProvider() 
+                                    => new ServiceCollection().AddInMemoryEventStream().BuildServiceProvider();
+        static EventStream CreateEventStream(IServiceProvider serviceProvider = null) 
```

Comment:
I'm typically explicit with declaration scope, just because people are so bad at remember the simple rules. :D